SwiftUIHomeworks/SwiftUIHomeworks/Custom Views/CustomSliderView.swift

```diff
@@ -0,0 +1,120 @@
+// CustomSliderView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Custom slider view
+struct CustomSliderView: View {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let opacityPurpleValue: CGFloat = 0.2
+        static let widthCircleValue: CGFloat = 30
+        static let widthTotalRectangleValue: CGFloat = 80
+        static let offsetYSliderValue: CGFloat = -80
+        static let widthLineCircleValue: CGFloat = 5
+        static let securityPointValue: CGFloat = 8
+        static let dollarTitle = "$"
+        static let oneStepValue = 500
+        static let oneStepCGFloatValue: CGFloat = 500
+        static let sizeFontCurrentStepValue: CGFloat = 10
+        static let offsetYCircleValue: CGFloat = 15
+        static let totalCGFloatValue: CGFloat = 5500
+    }
+
+    // MARK: - Public properties
+
+    @State var minValue: CGFloat
+    @State var maxValue: CGFloat
```

Comment:
```suggestion
    @State var minValue: CGFloat
    @State var maxValue: CGFloat
```

/State во вью модель

Reply:
Не уверен что нужно делать вьюМодель для кастомных элементов, подожду, что скажет ментор

Reply:
оставь пока так